src/base/rss/rss_autodownloader.cpp

```diff
@@ -451,7 +451,7 @@ void AutoDownloader::store()
     m_savingTimer.stop();
 
     QJsonObject jsonObj;
-    foreach (auto rule, m_rules)
+    for (auto &rule : qAsConst(m_rules))
```

Comment:
`const auto &rule`?

Reply:
I thought i had tried that but guess not, thanks it works.